cql3/Cql.g

```diff
@@ -1168,7 +1162,7 @@ listPermissionsStatement returns [std::unique_ptr<list_permissions_statement> st
       { $stmt = std::make_unique<list_permissions_statement>($permissionOrAll.perms, std::move(r), std::move(role), recursive); }
     ;
 
-permission returns [auth::permission perm]
+permission returns [auth::permission perm = auth::permission{}]
```

Comment:
Sad that this silly initialization (basically setting an enum to zero, whatever this means, I guess it just takes the first value of the enum) helps anything. As I said, I don't know why it helps and would have liked to know :-(